lib/seahorn/UfoOpSem.cc

```diff
@@ -1206,6 +1206,17 @@ void UfoOpSem::execBr(SymStore &s, const BasicBlock &src, const BasicBlock &dst,
   }
 }
 
+void UfoOpSem::execRange(SymStore &s, const llvm::BasicBlock::iterator begin,
+                         const llvm::BasicBlock::iterator end, ExprVector &side,
+                         Expr act) {
+  OpSemVisitor v(s, *this, side);
+  v.setActiveLit(act);
+  for (const auto &I : boost::make_iterator_range(begin, end)) {
```

Comment:
use `llvm::make_range` instead of `boost`

Reply:
Fixed

Reply:
Fixed.